app/routes/apply.role-details.tsx

```diff
@@ -0,0 +1,19 @@
+import { type MetaFunction } from "@remix-run/node";
```

Comment:
in this pr or a follow up pr, lets make it be apply/role-details/software-engineer or apply/role-details/product-manager etc. You can do this with remix dynamic segments, https://remix.run/docs/en/main/file-conventions/routes#dynamic-segments